packages/typescript/ai/tests/stream-to-response.test.ts

```
@@ -224,7 +224,7 @@ describe('toServerSentEventsStream', () => {
   it('should not send error if aborted', async () => {
     const abortController = new AbortController()
 
-    async function* errorStream(): AsyncGenerator<StreamChunk> {
+    async function* errorStream(): AsyncGenerator<any> {
       abortController.abort()
       throw new Error('Stream error')
     }
```

ESLint

[error] This generator function does not have 'yield'.

(require-yield)